packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2222,7 +2226,9 @@ export class ProjectService extends BaseService {
             fileUrl,
             valuesColumns: groupByColumns
                 ? Array.from(valuesColumnReferences)
-                : valuesColumns.map((col) => col.reference),
+                : valuesColumns.map(
+                      (col) => `${col.reference}_${col.aggregation}`,
```

Comment:
so that we have a unique identifier for a column that has an aggregation applied. 

For example:
- event
- event_sum

allows us to use both columns but on different axes, on different series, etc